app/daemon/daemon.go

```diff
@@ -318,12 +330,12 @@ func (d *daemon) createGame(tag, name string, sceanarioNo int) error {
 		return err
 	}
 
-	if err := env.StartGame(context.TODO(), tag, name, sceanarioNo); err != nil {
+	if err := env.Start(context.TODO(), tag, name, sceanarioNo); err != nil {
```

Comment:
Should we maybe take the context from the function call instead of having multiple contexts

Reply:
Yeah, it was for testing some stuff

Reply:
Super lets leave it for now